dvdrip.py

```diff
@@ -370,6 +371,8 @@ def MakeTitle(name, number, info):
                         warn("Cannot parse scan of title %d." % i)
 
     def Eject(self):
+        if os.name == 'nt':
+            return
```

Comment:
Apparently you can eject a disk on Windows from Python like this:

``` python
ctypes.windll.WINMM.mciSendStringW('open L: type CDAudio alias L_drive', None, 0, None)
```

I don't have a Windows machine to test this on, however. Also, this presumably ejects `L:`, and I don't know what the significance of "L_drive" is, or how to properly generalize it. So I'm not sure if you want to try that, or just disable ejecting on Windows entirely as you have it.

Reply:
I think the "L_drive" name can be chosen freely, it is just an alias for the next command. But I am not 100% sure, so I replaced `L` with the drive letter in both instances. 
There was another command needed to actually open the dvd tray then.

It works on my machine and opens the correct tray. If it fails, it won't not throw an exception, it just returns a non-zero value, so I guess it's safe to add.